### DISCUSSION TITLE
Windows Defender

### DISCUSSION THREAD

**Discussion by Stein-N** [memstechtips/UnattendedWinstall#192] [Q&A]
Is the windows Defender working normaly after using the unattended file or it it removing some parts of it?

**Comment by amordesoul** [comment]
I did one installation where I was able to enable Defender immediately after the installation finished. On my second installation the strange and janky app that pops up after installation completes that allows you to enable UAC and Defender crashed. Now I can't get it enabled. I've tried a few things from services and the registry.

I'm thinking of just running Malwarebytes instead.

**Comment by memstechtips** [comment]
Only the services of Windows Defender were disabled, it wasn't removed. You could use the UWScript.ps1 file to re-enable it.
